fili-core/src/main/java/com/yahoo/bard/webservice/table/PhysicalTable.java

```diff
@@ -58,10 +60,12 @@ public PhysicalTable(
         this.workingIntervals = Collections.synchronizedMap(new HashMap<>());
         this.logicalToPhysicalColumnNames = Collections.unmodifiableMap(logicalToPhysicalColumnNames);
         this.physicalToLogicalColumnNames = Collections.unmodifiableMap(
-                this.logicalToPhysicalColumnNames
-                        .entrySet()
-                        .stream()
-                        .collect(StreamUtils.toLinkedMap(Map.Entry::getValue, Map.Entry::getKey))
+                this.logicalToPhysicalColumnNames.entrySet().stream().sequential().collect(
```

Comment:
I believe `sequential` just forces the stream to be sequential, even if it was originally parallel. However, by default the stream of a set is sequential, so `sequential` doesn't actually do anything. 

So yes, I would suggest removing the sequential.